src/server.ts

```diff
@@ -0,0 +1,14 @@
+// @ts-ignore
```

Comment:
What's the error here? It's best practice to avoid such errors

Reply:
All right, I fixed it thanks